game-core/src/main/java/games/strategy/triplea/odds/calculator/DummyDelegateBridge.java

```diff
@@ -0,0 +1,140 @@
+package games.strategy.triplea.odds.calculator;
+
+import java.util.List;
+import java.util.Properties;
+
+import games.strategy.engine.data.Change;
+import games.strategy.engine.data.CompositeChange;
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.data.PlayerID;
+import games.strategy.engine.data.Unit;
+import games.strategy.engine.data.UnitHitsChange;
+import games.strategy.engine.delegate.IDelegateBridge;
+import games.strategy.engine.display.IDisplay;
+import games.strategy.engine.gamePlayer.IRemotePlayer;
+import games.strategy.engine.history.DelegateHistoryWriter;
+import games.strategy.engine.history.IDelegateHistoryWriter;
+import games.strategy.engine.random.IRandomStats;
+import games.strategy.engine.random.PlainRandomSource;
+import games.strategy.sound.HeadlessSoundChannel;
+import games.strategy.sound.ISound;
+import games.strategy.triplea.delegate.MustFightBattle;
+import games.strategy.triplea.ui.display.HeadlessDisplay;
+import games.strategy.triplea.ui.display.ITripleADisplay;
+
+/**
+ * Delegate bridge implementation with minimum valid behavior.
+ */
+public class DummyDelegateBridge implements IDelegateBridge {
```

Comment:
This looks very similar to the nested class of the same name in `OddsCalculator`.  Are they identical?  If so, can usage of the nested class be replaced with this one?

Ditto for `DummyGameModifiedChannel` and `DummyPlayer`.

Reply:
@ssoloff I'm actually pretty sure this was the intention.
I was asking myself the same question when briefly skimming the diff this morning.

Reply:
Indeed, this was conflicted before I submitted it. My mistake to not realize that it was badly resolved.

The PR should make more sense now with the inner classes dropped: a83e53f
Noting from the overview: " it is 95% moved code without further alterations.", now we see 'moved' code and not just new code...